geode-docs/basic_config/data_regions/region_naming.html.md.erb

```diff
@@ -24,7 +24,7 @@ follow these region naming guidelines.
 
 -   Permitted characters within region names are alphanumeric characters
 (`ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789`),
-the underscore character (`_`), and the hyphen character (`-`).
+the dot character (`.`), the underscore character (`_`), the square brackets characters (`[, ]`) and the hyphen character (`-`).
```

Comment:
Good clarification. An editorial suggestion: Since you've already established that you're talking about characters, you can simplify the language for readability without loss of precision.
Please change "the dot character" to "period", "the underscore character" to "underscore", "the square brackets characters" to "square brackets" and "the hyphen character" to "hyphen". Also for readability, "Permitted characters within region names" could be "Characters permitted in region names are".
The result would be:
Characters permitted in region names are alphanumeric characaters (`ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789`), period (`.`), underscore (`_`), square brackets (`[, ]`) and hyphen (`-`).

Reply:
Done! Thanks @davebarnes97!